modules/Hoot/managers/api.js

```diff
@@ -979,7 +982,7 @@ export default class API {
             });
     }
 
-    openDatasetInJosm( id, name ) {
+    openDatasetInJosm( id, name, outputName ) {
```

Comment:
this signature would feel better to me as `(id, name, ext)` and then the ext could be appended on line 996 below.